account_payment_term_extension/models/account_payment_term.py

```diff
@@ -178,16 +178,16 @@ def compute(self, value, date_ref=False):
                 next_date += relativedelta(days=line.days,
                                            weeks=line.weeks,
                                            months=line.months)
-            elif line.option == 'fix_day_following_month':
+            elif line.option == 'after_invoice_month':
                 # Getting 1st of next month
                 next_first_date = next_date + relativedelta(day=1, months=1)
                 next_date = next_first_date + relativedelta(days=line.days - 1,
                                                             weeks=line.weeks,
                                                             months=line.months)
-            elif line.option == 'last_day_following_month':
+            elif line.option == 'day_following_month':
                 # Getting last day of next month
                 next_date += relativedelta(day=31, months=1)
```

Comment:
@pedrobaeza Are you sure? last_day_following_month seems to work fine for us

Reply:
Does it take the specific day you have put? Seeing this code, I doubt it...

Reply:
@pedrobaeza Testing in runbot. It seems to work fine

Reply:
Yes, because in runbot now the code version is https://github.com/OCA/account-invoicing/pull/513/commits/2681c67346ffc4b2519cd646430e38b8ee817ccd#diff-d0188f250f37daab85f1680d28b455ccR193